radsat-sk/framework/fileio/RFram.c

```diff
@@ -28,6 +28,14 @@ void framInit(void) {
  * @return 0 for success, below 0 for failure. See hal/Storage/FRAM.h for more details.
  */
 int16_t framRead(uint8_t* data, uint32_t address, uint32_t size) {
+
+	uint16_t err = FRAM_read(data, address, size);

```

Comment:
Again, please review pointer or we can talk about how pointers work because the parameter passed into FRAM_read is not what its expecting